app/uk/gov/hmrc/uknwauthcheckerapi/models/eis/EisAuthTypes.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.uknwauthcheckerapi.utils
+package uk.gov.hmrc.uknwauthcheckerapi.models.eis
 
 object EisAuthTypes {
   val nopWaiver = "UKNW"
```

Comment:
As authType we moved to config, this object can be removed

Reply:
Also, LocalDateService can be removed as doesn't have any logic

Reply:
1. TestData uses EisAuthTypes in it's trait so it can't be removed.
2. I've replaced it's usage in places where i could, with appConfig.authType
3. IntegrationFrameworkService is using LocalDateService